kdumpctl

```diff
@@ -1030,6 +1032,53 @@ check_final_action_config()
 	esac
 }
 
+LUKS_SYSFS=/sys/kernel/crash_dm_crypt_keys
+reuse_loaded_luks_keys()
+{
+	local _luks_dev _count _cmds _cmd _state
+	declare -a _luks_devs
+
```

Comment:
On second thought, wouldn't it be safer to simply drop this function and always setup the a new key ring when loading the kdump kernel. It's more expensive but otherwise you could end up in a situation like
```
# dump to luks dev1
$ kdumpctl start
# edit kdump.conf to dump to luks dev2
$ kdumpctl rebuild
$ kdumpctl reload
```
which, IIUC, should fail as the key of dev1 is now used to decrypt dev2, which  most likely should fail.